src/utilities/new_versions.jl

```diff
@@ -280,24 +289,68 @@ function make_pr_for_new_version(
             if commit_was_success
                 @info("Commit was a success")
                 api_retry() do
-                    @mock git_push("origin", new_branch_name; force=true, env=env)
+                    @mock git_push(
+                        "origin", new_branch_name, pkey_filename; force=true, env=env
+                    )
                 end
 
-                create_new_pull_request(
+                new_pr, _ = create_new_pull_request(
                     forge,
                     repo,
                     new_branch_name,
                     master_branch_name,
                     new_pr_title,
                     new_pr_body,
                 )
+
+                cc_user && cc_mention_user(forge, repo, new_pr; env=env)
+                unsub_from_prs && unsub_from_pr(forge, new_pr)
             end
         end
     end
 
     return nothing
 end
 
+function cc_mention_user(
+    api::GitHub.GitHubAPI, repo::GitHub.Repo, pr::GitHub.PullRequest;
+    env=ENV,
+)
+    username = env["GITHUB_ACTOR"]
+    body = "cc @$username"
+
+    return @mock GitForge.create_pull_request_comment(
+        api,
+        repo.owner.login,
+        repo.name,
+        pr.id;
+        body=body,
+    )
+end
+
+function cc_mention_user(
+    api::GitLab.GitLabAPI, repo::GitLab.Project, pr::GitLab.MergeRequest;
+    env=ENV,
+)
+    username = env["GITLAB_USER_LOGIN"]
+    body = "cc @$username"
+
+    return @mock GitForge.create_pull_request_comment(
+        api,
+        repo.id,
+        pr.iid;
+        body=body,
+    )
+end
+
+function unsub_from_pr(api::GitHub.GitHubAPI, pr::GitHub.PullRequest)
+    return GitForge.unsubscribe_from_pull_request(api, pr.repo.id, pr.id)
+end
+
+function unsub_from_pr(api::GitLab.GitLabAPI, pr::GitLab.MergeRequest)
+    return @mock GitForge.unsubscribe_from_pull_request(api, pr.project_id, pr.iid)
```

Comment:
Just confirming `pr.iid` is a thing and not a typo of `pr.id`

Reply:
yup, iid is internal id